Handler/Cancel.hs

```diff
@@ -0,0 +1,22 @@
+module Handler.Cancel where
+
+import Import
+import Stripe
+
+postCancelR :: Handler Html
+postCancelR = do
+    plan <- runDB $ getBy404 $ UniquePlan freePlanId
+    Entity userId user <- requireAuth
+
+    result <- runStripe $ cancelSubscription $ userStripeId user
+
+    case result of
+        Just _ -> do
+            runDB $ update userId [UserPlan =. entityKey plan]
+
+            setMessage "Plan changed!"
```

Comment:
The UI terminology is slightly off from the backend. To the user they're *changing* from Business to Personal. To us, we're *cancelling* Business.

> Does carnival use I18n?

Not yet, but that's an enhancement I'd like to make some day.